payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -195,4 +271,11 @@ internal class CardNumberEditableController constructor(
     override fun onFocusChange(newHasFocus: Boolean) {
         _hasFocus.value = newHasFocus
     }
+
+    override fun onDropdownItemClicked(item: TextFieldIcon.Dropdown.Item) {
+        chosenBrand.value = when (val brand = CardBrand.fromCode(item.id)) {
+            CardBrand.Unknown -> null
```

Comment:
What’s the reason that we can’t set it to `CardBrand.Unknown`?

Reply:
That makes sense, but I’m wondering if we want to do the `brand.takeIf { it  != CardBrand.Unknown }` further down in the stack/closer to the network call. Mostly wondering because I’d want to “remember” a user’s selection: Say they enter `4000 0025`, select Visa, then remove the last digit and enter it again. We should auto-select Visa again as soon as its an available network, as the user has already expressed this intention before.

This is not something that we need to implement in this pull request, but I’d like to flag it for later.